### PR TITLE
added unycraft network to servermedia

### PR DESCRIPTION
**Pull request by danzelgeb**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by danzelgeb** [comment]
added unycraft network

File: minecraft_servers/unycraft/background.png
Comment (DoJapHD) [inline minecraft_servers/unycraft/background.png:1]: Compress

File: minecraft_servers/unycraft/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/unycraft/background@2x.png:1]: Compress

File: minecraft_servers/unycraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "server_name": "unycraft",
+    "nice_name": "UnyCraftNET",
+    "direct_ip": "unycraft.net",
+    "social": {
+        "web": "https://unycraft.net",
+        "discord": "https://dsc.gg/unycraft"
```
Comment (DoJapHD) [inline minecraft_servers/unycraft/manifest.json:7]: Please use https://discord.gg/[INVITE]

File: minecraft_servers/unycraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "server_name": "unycraft",
+    "nice_name": "UnyCraftNET",
+    "direct_ip": "unycraft.net",
+    "social": {
+        "web": "https://unycraft.net",
```
Comment (DoJapHD) [inline minecraft_servers/unycraft/manifest.json:6]: Not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is currently in maintenance;
Reopen PR once server is open to public

**Pull request closed without merging**